### ISSUE TITLE
Possibility of debug in local ruby execution

### ISSUE THREAD

**Issue by jkburges** [serverless/serverless#5708]
Consider a ruby serverless function such as:

```
require 'pry' 

def handler(event:, context:)
  binding.pry
end
```

When executing it locally, I see something like the following, however execution does _not_ pause at the breakpoint, rather it continues and the function runs to completion:

```
$ sls invoke local -f myFunction

From: /path/functions/my_function.rb @ line 4 Object#handler:

     3: def handler(event:, context:)
 =>  4:   require 'pry'; binding.pry
     5: end


[1] pry(Kernel)> 

$
```

I guess it's got something to do with the way node is launching the ruby process. Is it possible to pause at the breakpoint does anyone know? This would be a great feature in development mode for those of us used to using a ruby debugger.

Tagging @dholdren as the author of https://github.com/serverless/serverless/pull/5559 - maybe you might know?

**Comment by dschep** [comment]
My guess is that this is similar to the TTY issue I faced with python: 

https://github.com/serverless/serverless/blob/ad5decb080a7aa92cac1bc0f8632e3e57b99fe46/lib/plugins/aws/invokeLocal/invoke.py#L76-L82

**Comment by jkburges** [comment]
Thanks @dschep, attaching to tty worked: https://github.com/serverless/serverless/pull/5718

**Comment by pmuens** [comment]
Closing since @jkburges implemented this in #5718 🎉
